### ISSUE TITLE
review english terminolory

### ISSUE THREAD

**Issue by supermihi** [supermihi/karlchencloud#25]
found this one: https://github.com/Doppelkopf/Terminology
@eikel

**Comment by eikel** [comment]
Apart from small differences, we are using the same terminology. Do you think we should change something?

**Comment by supermihi** [comment]
No, I just found that list and thought it could be of help. Did not go through it in detail. :-)

Am 24. April 2021 13:00:31 MESZ schrieb Benjamin Eikel ***@***.***>:
